matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/media/UrlsExtractor.kt

```diff
@@ -35,7 +36,14 @@ internal class UrlsExtractor @Inject constructor() {
                             || it.msgType == MessageType.MSGTYPE_NOTICE
                             || it.msgType == MessageType.MSGTYPE_EMOTE
                 }
-                ?.body
+                ?.let { messageContent ->
+                    if (messageContent.relatesTo?.inReplyTo?.eventId != null) {
```

Comment:
Not sure if relevant but usually i use getRelationContent() extension, it handles encrypted relation

Reply:
OK, thanks, I will check